docs/OMS-Agent-for-Linux.md

```diff
@@ -186,6 +204,57 @@ SHARED_KEY=<Shared Key>
 `sudo /opt/microsoft/omsagent/bin/omsadmin.sh`
 4.	The file will be deleted on successful onboarding
 
+## Onboard a secondary workspace
+From 1.3.0-1, OMSAgent supports to onboard the agent to multiple workspaces.
+Run the omsadmin.sh command supplying the workspace id and key for your workspace, and -m to indicate secondary workspace:
+```
+cd /opt/microsoft/omsagent/bin
+sudo ./omsadmin.sh -w <WorkspaceID> -s <Shared Key> -m <Multi-Homing Marker>
+```
+NOTE: Secondary workspace is currently unable to pull the configuration from OMS service. We are working on it.
+
+## Onboard a secondary workspace using a file
+Reference [Onboarding using a file](#onboarding-using-a-file)
+Add the following line into /etc/omsagent-onboard.conf
+```
+MULTI_HOMING_MARKER=<Any marker you choose>

```

Comment:
What is an example of a marker you could choose? I'm not sure you provide an example in this doc.